### DISCUSSION TITLE
"make generate" failing with error

### DISCUSSION THREAD

**Discussion by kommuri-bharath** [prometheus/snmp_exporter#1296] [Q&A] [answered]
Hi Team,

I am getting below error when issues "make generate" command.

# make generate
make: git: No such file or directory
go build
>> Downloading Kemp LM MIBs to /tmp/tmp.7p8kmMM4x1
curl: (35) OpenSSL SSL_connect: Connection reset by peer in connection to kemptechnologies.com:443
make: *** [Makefile:205: mibs/.kemp-lm] Error 35

This is happening only when I download the "Source Code" files with versions 0.24.0 & 0.24.1.

When tried the same with "Source Code" data downloaded with 0.20.0 "make generate" works absolutely fine.

Please let me know what is the fix. (Unfortunately my server doesn't have access to internet)

Thanks in advance !!

**Comment by SuperQ** [comment, marked as answer]
Access to the internet is required for `make mibs`. You will have to build this outside of your server.

**Reply by kommuri-bharath** [reply]
Hi,

Thanks for the update.

But when I use "Source Code" data downloaded with 0.20.0 the "make generate" is working absolutely fine.

Could you please let me know how can I create a generator using the 0.24.1 code offline? Do I need to download the mibs files before hand?

"You will have to build this outside of your server" --> How can I do that without internet?

Thanks in advance !
